front/se_project/src/components/logout.jsx

```diff
@@ -0,0 +1,14 @@
+import React, {Component} from 'react';
```

Comment:
check does it works with back?
check with cookie

Reply:
ok